src/core_modules/capture-core/components/WidgetProfile/WidgetProfile.component.js

```diff
@@ -20,6 +20,8 @@ import {
     useTeiDisplayName,
 } from './hooks';
 import { DataEntry, dataEntryActionTypes, TEI_MODAL_STATE, convertClientToView } from './DataEntry';
+import { OverflowButton } from '../Buttons';
```

Comment:
I didn't think `WidgetProfile` is self-contained though? Or should we try to keep it more separated either way?

Reply:
I think it's mainly the profile widget and enrollment widget that causes people to talk about self-containedness. What do you think @eirikhaugstulen?

Reply:
This is fine. We will be using components/functions from other parts of the app, we will just have to make sure we include these when we finally make a library. Most important part (in a "self-contained" component) is that we don't use the metadata memory objects since refactoring such code would be really time consuming.